_maps/configs/srm_rigger.json

```diff
@@ -0,0 +1,25 @@
+{
+	"map_name": "Saint-Roumain Modified Rigoureux-class Auxiliary Cutter",
+	"map_short_name": "Rigoureux-class",
```

Comment:
You should add lines for prefix and ship namelists in this section.

Reply:
SRV?

Reply:
SRV or SRSV would work well for prefix, yeah. Take your pick for namelists, though we don't have a suitably religious one. Maybe use the mythological and beasts lists.